packages/remote-config/ios/RNFBConfig/RNFBConfigModule.m

```diff
@@ -223,6 +247,41 @@ + (BOOL)requiresMainQueueSetup {
   }
 }
 
+RCT_EXPORT_METHOD(onConfigUpdated
+                  : (FIRApp *)firebaseApp
+                  : (RCTResponseSenderBlock)callback) {
+    FIRConfigUpdateListenerRegistration *newRegistration = [[FIRRemoteConfig remoteConfigWithApp:firebaseApp] addOnConfigUpdateListener:^( FIRRemoteConfigUpdate * _Nonnull configUpdate, NSError * _Nullable error) {
+        NSMutableDictionary *userInfo;
+
+        if (error != nil) {
+            userInfo = [NSMutableDictionary dictionary];
+
+            [userInfo setValue:@(NO) forKey:@"fatal"];
+            [userInfo setValue:@"unknown" forKey:@"code"];
+            [userInfo setValue:error.localizedDescription forKey:@"message"];
+            [userInfo setValue:@(error.code) forKey:@"nativeErrorCode"];
+            [userInfo setValue:error.localizedDescription forKey:@"nativeErrorMessage"];
+        }
+
+        callback(@[
+            @{
+                @"updatedKeys" : [configUpdate.updatedKeys allObjects],
+            },
```

Comment:
if there is an error, I'm not sure we should be returning the first value, it should probably be `[NSNull null]`.

Reply:
Fixed this